app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -77,6 +77,19 @@ def ping
       render json: { status: 'OK' }
     end
 
+    api :PUT, "/organizations/:organization_id/toggle_simple_content_access",
+      N_("Toggle simple sontent access for a manifest")
+    param :organization_id, :number, :desc => N_("Organization ID"), :required => true
+    def toggle_simple_content_access
```

Comment:
what do you think about creating two APIs:

`/organizations/:id/simple_content_access/enable` and `organizations/:id/simple_content_access/disable` ?